website/docs/r/machine_learning_workspace.html.markdown

```diff
@@ -422,6 +424,18 @@ An `managed_network` block supports the following:
 
 ---
 
+An `serverless_compute` block supports the following:
+
+* `custom_subnet_id` - (Optional) The ID of an existing virtual network subnet in which serverless compute nodes should be deployed.
+
+* `no_public_ip_enabled` - (Optional) Is serverless compute nodes deployed in custom vNet would have no public IP addresses enabled for a workspace with private endpoint? Defaults to `false`.
+
+~> **Note:** Not supported to set `public_ip_enabled` to `false` without `subnet_id` when `public_network_access_enabled` is `false`.
+
+~> **Note:** Not supported to update `public_ip_enabled` from `true` to `false` when `subnet_id` is null or empty
```

Comment:
this needs to be updated

Reply:
delete the notes?

Reply:
to name the property `serverlessComputeNoPublicIP` to `public_ip_enabled`, value has been flipped. I add the constraint in the resource and also the doc.